test/js.go

```diff
@@ -0,0 +1,73 @@
+//go:build js
+
+package test
+
+import (
+	"context"
+	"syscall/js"
+	"testing"
+
+	"github.com/cosmos/gogoproto/jsonpb"
+	"github.com/cosmos/gogoproto/proto"
+	"github.com/stretchr/testify/require"
+
+	acpjs "github.com/sourcenetwork/acp_core/pkg/js"
+	"github.com/sourcenetwork/acp_core/pkg/runtime"
+	"github.com/sourcenetwork/acp_core/pkg/types"
+)
+
+var _ types.PlaygroundServiceServer = (*PlaygroundJS)(nil)
+
+func newPlaygroundJSImpl(t *testing.T, manager runtime.RuntimeManager) *PlaygroundJS {
+	proxy, err := acpjs.NewPlaygroundServiceProxy(context.TODO(), manager)
+	require.NoError(t, err)
+
+	return &PlaygroundJS{
+		proxy: proxy,
+		this:  proxy.AsJSValue(),
+	}
+}
+
+type PlaygroundJS struct {
```

Comment:
fixme: in general, these methods should do a reverse map.

it should:
- receive go objects
- transform them to js objects
- call the handler which returns JS object
- transform the js objects back to go
- return the transformed objects